vcpkg.json

```diff
@@ -9,7 +9,7 @@
   "overrides": [
     {
       "name": "arrow",
-      "version": "17.0.0"
+      "version": "18.1.0"
```

Comment:
Can we also update the `builtin-baseline` to a more recent commit? Preferably one of the tagged releases (https://github.com/microsoft/vcpkg/releases)

This will pick up any updates to the dependencies required by arrow.

Reply:
Okay, I switched it to the latest release ([2025.01.13 Release](https://github.com/microsoft/vcpkg/releases/tag/2025.01.13))